game_patch/multi/server.cpp

```diff
@@ -554,6 +568,19 @@ static bool check_player_ac_status([[maybe_unused]] rf::Player* player)
     return true;
 }
 
+std::pair<bool, std::string> is_level_name_valid(std::string_view level_name_input)
```

Comment:
Two things:
* I think this name is not good - validation of something (name in this case) usually means to check if it follows some rules, like not having denied characters, having a proper length, etc. In this case it's not really a validation but checking for existence. The name `does_level_exist` would be more clear
* I think this function has too broad responsibility. I would rather prefer a function that adds extension (not necessary rfl) if it's missing (may be useful in other contexts) and another to check existence (this one is probably not needed because it's a single line). To be honest all that code could be inlined in the command too. It will be trivial after you use `string_ends_with`.

Reply:
Valid point (lol) on the use of the word `valid`. Something like `is_level_loaded` might fit a little better than `does_level_exist` though - WDYT?

> I would rather prefer a function that adds extension (not necessary rfl) if it's missing (may be useful in other contexts)

I like this idea - I don't know of another use case off the top of my head but I could see it coming up down the road.

Reply:
`is_level_loaded` is kind of confusing. `does_level_exist` is imo much better.

Reply:
is_level_loaded doesn't make sense. Loaded level is the current one. For example to check if any is loaded we use `rf::level.flags & rf::LEVEL_LOADED`.
Idk why `does_level_exist` is bad, but alternative could be `is_level_available`